src/Microsoft.Dnx.Runtime/NuGet/SemanticVersion.cs

```diff
@@ -16,9 +16,6 @@ public sealed class SemanticVersion : IComparable, IComparable<SemanticVersion>,
         public SemanticVersion(string version)
             : this(Parse(version))
         {
-            // The constructor normalizes the version string so that it we do not need to normalize it every time we need to operate on it. 
-            // The original string represents the original form in which the version is represented to be used when printing.
-            OriginalString = version;
```

Comment:
I believe the OriginalString is only used for storing the unnormalized version number and was only used for determining whether normalization occurred. From our previous conversation, this functionality will no longer be needed so isn't this field now redundant?